.reuse/dep5

```diff
@@ -90,3 +90,8 @@ Copyright: 2020-2021, Olga Khlopkova <mesozoic.drones@gmail.com>
            2020, Vladimir Byko-Ianko <bykoianko@gmail.com>
            2020, mesozoic-drones <mesozoic.drones@gmail.com>
 License: MIT
+
+Files: cdn/*
+Copyright: 2025, Veniamin Gvozdikov
+           2025, Organic Maps (formerly OMaps) contributors
+License: AGPL-3.0-or-later
```

Comment:
All the changes in repo are covered by default with the entry:
```
Files: *
Copyright: 2020, My.com B.V. (Mail.Ru Group)
           2021-2025, Organic Maps Contributors
License: Apache-2.0
```
This dep5 file was auto generated (?) for 3party staff, no need to add personal copyrights here. Especially AGPL.

Reply:
This code is not directly related to the client code, and it introduces AGPL in the main repo, which is very confusing. It should be in a different repo.

AGPL does not allow customization of the code for specific needs, like Apache 2 or MIT licenses allow. This customization is the main attractive feature of OM now for many projects.